pkg/resource/acl/custom_set_output.go

```diff
@@ -0,0 +1,67 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//     http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package acl
+
+import (
+	"context"
+
+	svcsdk "github.com/aws/aws-sdk-go/service/memorydb"
+
+	svcapitypes "github.com/aws-controllers-k8s/memorydb-controller/apis/v1alpha1"
+	"github.com/aws-controllers-k8s/memorydb-controller/pkg/events"
+)
+
+func (rm *resourceManager) customDescribeACLsSetOutput(
+	ctx context.Context,
+	r *resource,
+	resp *svcsdk.DescribeACLsOutput,
+	ko *svcapitypes.ACL,
+) (*svcapitypes.ACL, error) {
+	if len(resp.ACLs) == 0 {
+		return ko, nil
+	}
+	err := rm.customSetOutputSupplementAPIs(ctx, r, ko)
+	if err != nil {
+		return nil, err
+	}
+	return ko, nil
+}
+
+func (rm *resourceManager) customSetOutputSupplementAPIs(
+	ctx context.Context,
+	r *resource,
+	ko *svcapitypes.ACL,
+) error {
+	events, err := rm.provideEvents(ctx, r.ko.Spec.Name, 20)
```

Comment:
Max results is not a required field on the `DescribeEvents` SDK operation, but I agree that we should not have *too many* events in the status or it will become very messy. Could you please make `20` into a constant in the new `pkg/events/events.go` file and add some context around why you chose 20.